aspnetcore/tutorials/razor-pages/model.md

```diff
@@ -244,11 +244,39 @@ In this section, the **Package Manager Console** (PMC) window is used to:
    ```powershell
    Add-Migration InitialCreate
    Update-Database
-
    ```
 
-# [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
-* In the **Solution Tool Window**, right-click the *RazorPagesMovie* project, and then select **Open in Terminal**. 
+The preceding commands:
+
+* The `migrations` command generates code to create the initial database schema. The schema is based on the model specified in `DbContext`. The `InitialCreate` argument is used to name the migrations. Any name can be used, but by convention a name is selected that describes the migration.
+
+* The `update` command runs the `Up` method in migrations that have not been applied. In this case, `update` runs the `Up` method in the `Migrations/<time-stamp>_InitialCreate.cs` file, which creates the database.
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+* Right-click the *RazorPagesMovie.csproj* project, and then select **Open in Integrated Terminal**.
+
+  The **Terminal** window opens with the command prompt at the project directory, which contains the Program.cs and .csproj files.
+
+* Run the following .NET CLI commands:
+
+  ```dotnetcli
+  dotnet ef migrations add InitialCreate
+  dotnet ef database update
+  ```
+
+> [!NOTE]
+> For SQLite, column type for the `Price` field is set to `TEXT`. This is resolved in a later step.
```

Comment:
I would move this note to after the command explanation bullets. Same situation in the macOS tab.